frontend/appflowy_flutter/lib/plugins/document/application/doc_bloc.dart

```diff
@@ -146,11 +146,11 @@ class DocumentBloc extends Bloc<DocumentEvent, DocumentState> {
     final result = await _documentService.openDocument(viewId: view.id);
     return result.fold(
       (l) => left(l),
-      (r) async => right(await _initAppFlowyEditorState(r)),
+      (r) async => right(await _initAppFlowyEditorState(r, view.id)),
```

Comment:
No longer needed right?
```suggestion
      (r) async => right(await _initAppFlowyEditorState(r)),
```